Django 2.2 test suite errors out when the selenium package
is not present. This way, it passes without any issue.

Not intended for upstream.

--- Django-2.2.9/tests/admin_inlines/tests.py
+++ Django-2.2.9/tests/admin_inlines/tests.py
@@ -1,4 +1,4 @@
-from selenium.common.exceptions import NoSuchElementException
+#from selenium.common.exceptions import NoSuchElementException
 
 from django.contrib.admin import ModelAdmin, TabularInline
 from django.contrib.admin.helpers import InlineAdminForm
